packages/muelu/src/Misc/MueLu_InverseApproximationFactory_def.hpp

```diff
@@ -71,11 +72,13 @@ namespace MueLu {
   template <class Scalar, class LocalOrdinal, class GlobalOrdinal, class Node>
   RCP<const ParameterList> InverseApproximationFactory<Scalar, LocalOrdinal, GlobalOrdinal, Node>::GetValidParameterList() const {
     RCP<ParameterList> validParamList = rcp(new ParameterList());
+    using Magnitude = typename Teuchos::ScalarTraits<Scalar>::magnitudeType;
 
     validParamList->set<RCP<const FactoryBase> >("A", NoFactory::getRCP(), "Matrix to build the approximate inverse on.\n");
 
     validParamList->set<std::string>            ("inverse: approximation type",  "diagonal", "Method used to approximate the inverse.");
-    validParamList->set<bool>                   ("inverse: fixing",              false     , "Fix diagonal by replacing small entries with 1.0");
+    validParamList->set<Magnitude>              ("inverse: drop tolerance",      1e-8      , "Below threshold values are dropped from the matrix.");
```

Comment:
```suggestion
    validParamList->set<Magnitude>              ("inverse: drop tolerance",      1e-8      , "Values below this threshold  are dropped from the matrix (or fixed if diagonal fixing is active).");
```

Reply:
Changed the description to the suggested one.